dojo/api_v2/serializers.py

```diff
@@ -1531,13 +1532,27 @@ def get_engagement(self, obj):
         )
 
     def validate(self, data):
+        findings = data.get('accepted_findings', [])
+        findings_ids = list(map(lambda x: x.id, findings))
```

Comment:
```suggestion
        findings_ids = [x.id for x findings]
```
List comprehension would be easier to read here and also has less datatype conversions